### ISSUE TITLE
LMobileNet, what does 'L' mean?

### ISSUE THREAD

**Issue by HaoLiuHust** [deepinsight/insightface#4]

**Comment by nttstar** [comment]
Means using stride11 in its first conv layer.

**Comment by HaoLiuHust** [comment]
@nttstar Thanks for you reply, I am training now. another question,

LMobileNetE | 0.99633+-0.00314
-- | --
is this model use sphereloss?

**Comment by nttstar** [comment]
Softmax only, if not specified. Overlap exists between vggface2 and lfw but in my current experiments they're not removed. You can compare the relative number if the performance gap is larger than 0.1. 
You can also check [https://github.com/happynear/FaceDatasets](url) for overlaps. I have 3 more runtime verification testing other than LFW and will be released soon.

**Comment by HaoLiuHust** [comment]
Thanks for your work, so sphereloss did not get better than softmax in your experiment?

**Comment by nttstar** [comment]
It depends on your network and dataset. Now I can not get sphere working for se-resnet50@vggface2, but it works for another dataset.  If you can make it work on vggface2, please let me know.

**Comment by HaoLiuHust** [comment]
I am training LMobileNetE@vggface2 with sphereloss, still finetuning the network for now, I can only get about 98.8% acc on lfw, I will try more

**Comment by nttstar** [comment]
I can get 99.6% on LMobileNetE@vggface2@sphere and better acc of other metrics than softmax.

**Comment by HaoLiuHust** [comment]
what's your training params? I have clean the vggface2 at first, and for now I can only get 98.8 use "adam" optimizer, beta = 1000, margin = 4, scale = 0.9993,
                             beta_min = 4., verbose = 1000

**Comment by nttstar** [comment]
You can try yourself as I did not clean the data.  Commonly I use SGD optimizer with mom=0.9 and lr=0.1, wd=5.e-4.  beta = 1000, beta_min = 5.  lr decrease to 0.01 and 0.001 after some epochs.

**Comment by HaoLiuHust** [comment]
for sphereloss, the evaluation metric is still euclid distance?

**Comment by nttstar** [comment]
Yes. And there should be no difference between euc distance and cosine sim if you normalize the vector firstly,

**Comment by HaoLiuHust** [comment]
Thanks, your input shape is 112x112? have you try 112x96？ since you mentioned in the description

**Comment by nttstar** [comment]
112x112 is slightly better than 112x96 on large pose variation

**Comment by nerddd** [comment]
@nttstar What's the reference points of 112*112  when aligning face?Thanks

**Comment by nttstar** [comment]
@nerddd +8 from the points of 112*96
